README.md

```diff
@@ -35,7 +35,7 @@ When you want to run the project again (and you will, it's great):
 
 ### Environment Variables
 
-Your `.env` file should contain the following items in the format: _key_=_value_
+Your `.env` file should contain the following items in the format: _key_=_"value"_
```

Comment:
Contributing code _and_ fixing our docs, you're making the world a better place

Reply:
Why the quotes? I don't have them in my `.env` file.

Reply:
It wouldn't build on my machine without them

Reply:
What OS are you running?

Reply:
(Although it shouldn't matter as we are using containers)
@Pinpickle What do you do?

Reply:
MacOS at the moment as I couldn't get Swift to install on Linux

Reply:
Oh, are you running the project in docker?

Reply:
I'm using docker on my Mac, but I'll try later on my laptop